static/js/components/CommentTree_test.js

```diff
@@ -125,11 +125,23 @@ describe("CommentTree", () => {
 
   it('should include a "reply" button', () => {
     const wrapper = renderCommentTree()
-    wrapper.find(".comment-action-button").at(0).simulate("click")
+    wrapper.find(".comment-action-button.reply-button").at(0).simulate("click")
     assert.ok(beginEditingStub.called)
     assert.ok(beginEditingStub.calledWith(replyToCommentKey(comments[0])))
   })
 
+  it('should not include a "reply" button for deleted posts', () => {
+    comments[0].deleted = true
+    const wrapper = renderCommentTree()
+    assert.ok(
+      wrapper
+        .find(".comment")
+        .at(0)
+        .find(".comment-action-button.reply-button")
+        .exists()
+    )
+  })
```

Comment:
the issue with this is that comments[0].replies has undeleted comments in it, so there will still be a .reply-button somewhere in the tree. I think just setting comments[0].replies = [] should make the test fail (as it should!)

Reply:
I fixed it instead by altering the selector.

Reply:
I thought I had pushed this but apparently didn't since I rebased and didn't force push it.